backend/src/main/java/com/woowacourse/levellog/team/presentation/TeamController.java

```diff
@@ -56,6 +56,13 @@ public ResponseEntity<Void> update(@PathVariable final Long teamId,
         return ResponseEntity.noContent().build();
     }
 
+    @PostMapping("/{teamId}/close")
+    public ResponseEntity<Void> close(@PathVariable final Long teamId,
```

Comment:
```suggestion
    @PutMapping("/{teamId}/close")
    public ResponseEntity<Void> close(@PathVariable final Long teamId,
```
갑자기 생각난건데 close가 되는게 왜 POST 메서드인가요,,? 🤔🤔
Team 속성의 일부가 수정되기 때문에 `PUT`이 더 적합하다고 생각합니다.
#161

Reply:
우선 저는 POST 파입니다

값을 바꾸지만 내가 원하는 값을 요청으로 주고 그대로 바꾸는게 아니라 어떤 행위를(팀을 종료하는) 하게하는 요청이기 때문입니다

API 구현을 하다보면 CRUD가 아닌 어떤 행위를 시키기도 하는데 그런 경우 저는 POST를 쓰는 편이에요
(post는 생성뿐 아니라 업데이트도 한다고해서...)

그리고 get put delete는 뭔가 그 특징이 있는것 같아서 어떤 행위를 표현하기 적당하지 않다고 생각했어요

예를들면 delete는 삭제만을 위해서 만들어졌기 때문에 바디를 담으면 안된다거나(지금은 넣을 수는 있지만 예전에는 에러가 발생했다고함)
get은 캐싱이 된다거나 브라우저에 기록이 남는다거나
하는 특징들이요 !

Reply:
```http
PUT /api/teams/{teamId}/status
{
    "isClosed": true
}
```
위와 같은 방식으로 url에서 동사를 지울 수 있겠네요.
POST가 업데이트도 한다는 부분은 조금 납득이 어렵네요..

참고: https://bcho.tistory.com/953

Reply:
POST가 주로 서버에 새 리소스 생성을 요청할 때 사용되긴 하지만, 꼭 새 리소스가 생성되지 않더라도 서버에서 뭔가 상태의 변화가 일어나야 하는 경우에도 사용되므로 저도 이 상황에서는 POST입니다. 멱등성 부분에서는 PUT이 괜찮을 거 같기도 한데, 팀 인터뷰 종료 API의 경우에는 값의 수정(update) 보다는 상태의 변화에 가깝게 느껴지네요👀

Reply:
![image](https://user-images.githubusercontent.com/76840965/182534639-595acebc-9601-4a14-9032-df9cd782746c.png)
출처 : 인프런 강의 모든 개발자를 위한 HTTP 웹 기본 지식 - 김영한

Reply:
👍